mutatest/cli.py

```diff
@@ -691,7 +702,9 @@ def main(args: argparse.Namespace) -> None:
         break_on_survival=run_mode.break_on_survival,
         break_on_error=run_mode.break_on_error,
         break_on_unknown=run_mode.break_on_unknown,
+        break_on_timeout=run_mode.break_on_timeout,
         ignore_coverage=args.nocov,
+        max_runtime=args.timeout_factor * clean_runtime_1.seconds,
```

Comment:
Notice that the `max_runtime` config is not passed directly as a command line argument, but is instead computed here.